config/config.ttl

```diff
@@ -1,14 +1,14 @@
-@base <https://github.com/solid/conformance-test-harness/> .
-@prefix test-harness: <https://github.com/solid/conformance-test-harness/> .
-@prefix solid-test: <https://github.com/solid/conformance-test-harness/vocab#> .
+base <https://github.com/solid/conformance-test-harness/>
+prefix test-harness: <https://github.com/solid/conformance-test-harness/>
+prefix solid-test: <https://github.com/solid/conformance-test-harness/vocab#>
 
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix doap: <http://usefulinc.com/ns/doap#> .
-@prefix earl: <http://www.w3.org/ns/earl#> .
-@prefix solid: <http://www.w3.org/ns/solid/terms#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix doap: <http://usefulinc.com/ns/doap#>
+prefix earl: <http://www.w3.org/ns/earl#>
+prefix solid: <http://www.w3.org/ns/solid/terms#>
+prefix dcterms: <http://purl.org/dc/terms/>
```

Comment:
Personally, I tend to prefer my prefixes as short as possible, so I tend to go for `dct` or even `dc` if there isn't any risk of confusion with the old namespace.

Reply:
I used dcterms as it was preferred by the Inrupt coding standards for vocabs

Reply:
Here is the mention of this: https://github.com/inrupt/public-documentation/blob/a54e74755cfbae3220acecbb1258b303d32a6e55/coding-conventions/rdf-vocabulary-guidelines.md#:~:text=Note%3A%20Anyone%20can,so%20do%20we!

Reply:
heh, I like the `@prefix a: b .` pattern :) I like the dot. The dot makes it look consistent. Also dcterms > dc > dct :)

but whatever..

Reply:
That's another change I made to conform with the vocab standards: https://github.com/inrupt/public-documentation/blob/a54e74755cfbae3220acecbb1258b303d32a6e55/coding-conventions/rdf-vocabulary-guidelines.md#prefixes. I had no strong opinion but when someone tells me "you should do it this way" and provides sound reasoning - who am I to disobey! ;)